Assets/Resources/UI/Icon.uss

```diff
@@ -0,0 +1,38 @@
+﻿/* The actual image node */
+.icon {
+    --icon-image: var(--icon-plus);
+    --icon-tint-color: var(--color-Blue700);
+    
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+    background-size: contain;
+    -unity-background-image-tint-color: var(--icon-tint-color);
+    background-image: var(--icon-image);
+    width: 32px;
+    height: 32px;
+
+    flex-shrink: 0;
+    flex-grow: 0;
+    margin-top: 0;
+    margin-right: 0;
+    margin-bottom: 0;
+    margin-left: 0;
+    padding-top: 0;
+    padding-right: 0;
+    padding-bottom: 0;
+    padding-left: 0;
+}
+
+.icon.tint-white { --icon-tint-color: var(--color-White); }
+.icon.tint-black { --icon-tint-color: var(--color-Black); }
+.icon.tint-blue-50 { --icon-tint-color: var(--color-Blue50); }
+.icon.tint-blue-100 { --icon-tint-color: var(--color-Blue100); }
+.icon.tint-blue-200 { --icon-tint-color: var(--color-Blue200); }
+.icon.tint-blue-300 { --icon-tint-color: var(--color-Blue300); }
+.icon.tint-blue-700 { --icon-tint-color: var(--color-Blue700); }
+.icon.tint-blue-900 { --icon-tint-color: var(--color-Blue900); }
+
+.icon.image-plus { --icon-image: var(--icon-plus); }
```

Comment:
This is a list of supported images for icons (you can still override them using the UI Builder) - and these will set the `--icon-image` USS variable above. Setting the variable instead of concrete properties helps keep it simple since I can do whatever I want in the USS above, by using variables this will never break.

I had hoped that we could have used a "map" (similar to how SCSS works) and get/set the list from the theme - but Unity's USS is not yet as advanced